SoftLayer/managers/hardware.py

```diff
@@ -255,12 +255,41 @@ def rescue(self, hardware_id):
 
             result = mgr.rescue(1234)
         """
-        return self.hardware.bootToRescueLayer(id=hardware_id)
+        return self.hardware.bootToRescueLayer(id=network_id)
 
-    def change_port_speed(self, hardware_id, public, speed):
+    def trunk(self, hardware_id, vlan_id):
+        """Add vlan trunk to a server.
+
+        :param integer hardware_id: the hardware ID to add vlan trunk.
+        :param integer vlan_id: the vlan id.
+
+        Example::
+
+            # Add trunk
+            result = mgr.trunk(824671, 1341234)
+            # Remove trunk
+            result = mgr.trunk(824671, -1341234)
```

Comment:
Although this is neat, it's not at all obvious that making the vlan id negative will remove it... How I would do this is make two methods: `add_trunk` and `remove_trunk` so it's obvious what kind of action is happening. In the CLI, you could have `--remove-trunk=1234` and `--add-trunk=4321` options.